.vscode/settings.json

```diff
@@ -7,7 +7,9 @@
     "**/generated/**": true
   },
   "rust-analyzer.check.allTargets": true,
+  "rust-analyzer.check.command": "clippy",
   "rust-analyzer.check.features": "all",
+  "rust-analyzer.checkOnSave": true,
```

Comment:
just thinking out loud, would `rust-analyzer.checkOnSave` be something that we want to enforce for all users of the repo? I wonder if we should leave it up to the user. It is not really affected by whether we are using `clippy` or not.

Not a blocker though.